cocotb/binary.py

```diff
@@ -35,6 +35,9 @@
 import warnings
 
 resolve_x_to = os.getenv('COCOTB_RESOLVE_X', "VALUE_ERROR")
+rand_gen = random.Random()
+rand_gen.seed(os.getenv('RANDOM_SEED', random.getrandbits(32)))
```

Comment:
The seeding of the random number generator is done in `__init__.py`, you can leave out those two lines.